server/middlewares/controllersValidator/ReqBodyValidator.js

```diff
@@ -0,0 +1,70 @@
+import Validator from '../../helpers/Validator';
+
+class ReqBodyValidator {
+  static validateFood(req, res, next) {
+    const {
+      name,
+      description,
+      price,
+      imageurl
+    } = req.body;
+
+    if (!name || !description || !price || !imageurl) {
+      return res.status(400).json({
+        message: 'Bad request, no field should not be missing',
+      });
+    }
+
+    if (!Validator.hasAminLength(name) || !Validator.hasAminLength(description)) {
+      return res.status(400).json({
+        message: 'Fields length must not be less than three',
+      });
+    }
+
+    if (!Validator.isNumber(price)) {
+      return res.status(400).json({
+        message: 'Price must be a number',
+      });
+    }
+
+    req.body = {
+      name: name.trim(),
+      description: description.trim(),
+      price: price.trim()
+    };
+    return next();
+  }
+
+  static validateOrder(req, res, next) {
+    const {
+      userid,
+      mealid,
+      quantity,
+      status
+    } = req.body;
+
+    if (!userid || !mealid || !quantity || !status) {
+      return res.status(400).json({
+        message: 'Bad request, no field should not be missing',
+      });
+    }
+
+
+    if (!Validator.isNumber(userid) || !Validator.isNumber(mealid)
```

Comment:
I think you should either extract this into a function or sperate the entries you are checking for, let the user know which field has the wrong input.

Reply:
Okay thanks. On it..